### DISCUSSION TITLE
In the board the status metadata field are not present to create the vue for the board

### DISCUSSION THREAD

**Discussion by mcudrio** [marcusolsson/obsidian-projects#668] [Help]
### What happened?

I have create a project from a folder, set these metadata :

```
tags:
aliases:
type:
category:
date: 2023-01-27 03:09:56
status:
  - process
```

In the table view I see all the metadata fields and the values in-it, but when I create and bord vue, the field status is not available, any idea ?

### What did you expect to happen?

I have delete the value in the field and write another word
I do the same in the sandbox and the problem is still here

![Screenshot 2023-08-23 at 02 45 34](https://github.com/marcusolsson/obsidian-projects/assets/5808634/18cd8798-30b1-46e6-9d4d-2b77da874104)

![Screenshot 2023-08-23 at 02 46 39](https://github.com/marcusolsson/obsidian-projects/assets/5808634/99e720e6-a14b-4336-801a-af54fa4b53cd)

![Screenshot 2023-08-23 at 02 47 02](https://github.com/marcusolsson/obsidian-projects/assets/5808634/5635b979-63a5-422a-8c18-4ada8f1cf168)

![Screenshot 2023-08-23 at 02 47 41](https://github.com/marcusolsson/obsidian-projects/assets/5808634/950175d7-e93c-4437-9ea4-2a7b086bab50)

![Screenshot 2023-08-23 at 02 48 00](https://github.com/marcusolsson/obsidian-projects/assets/5808634/d8cf61e0-0f45-4f64-bc6a-02f6c8b51013)

![Screenshot 2023-08-23 at 02 39 44](https://github.com/marcusolsson/obsidian-projects/assets/5808634/7952f06c-cbfe-4846-88bd-b542d7ad20a1)

![Screenshot 2023-08-23 at 02 49 16](https://github.com/marcusolsson/obsidian-projects/assets/5808634/74ee9d2d-3a0b-4543-a9e2-335c574cfa74)

![Screenshot 2023-08-23 at 02 50 44](https://github.com/marcusolsson/obsidian-projects/assets/5808634/0354b073-b552-45ee-9642-b5a8b098a723)

![Screenshot 2023-08-23 at 02 51 11](https://github.com/marcusolsson/obsidian-projects/assets/5808634/a8568496-e7d7-4877-802e-ca056cd416d8)

### How can we reproduce it (as minimally and precisely as possible)?

Create a notes
put the status metadata
and check if the project's board show the status metadata

### Anything else we need to know?

my config

SYSTEM INFO:
	Obsidian version: v1.3.7
	Installer version: v1.3.5
	Operating system: Darwin Kernel Version 22.5.0: Thu Jun  8 22:22:19 PDT 2023; root:xnu-8796.121.3~7/RELEASE_ARM64_T8103 22.5.0
	Login status: logged in
	Catalyst license: none
	Insider build toggle: off
	Live preview: on
	Legacy editor: off
	Base theme: light
	Community theme: AnuPpuccin v1.4.4
	Snippets enabled: 10
	Restricted mode: off
	Plugins installed: 72
	Plugins enabled: 59
		1: Outliner v4.8.0
		2: Dataview v0.5.56
		3: Tag Wrangler v0.5.11
		4: Kindle Highlights v1.9.2
		5: Better File Link v1.1.3
		6: Find orphaned files and broken links v1.9.0
		7: Hover External Link Plugin v1.0.0
		8: Natural Language Dates v0.6.1
		9: Theme Picker v1.0.8
		10: Timestamp Notes v1.0.8
		11: URL Namer v1.0.1
		12: Advanced Tables v0.19.1
		13: Diagrams v1.5.0
		14: TimeStamper v1.3.0
		15: Note Refactor v1.7.1
		16: Checklist v2.2.12
		17: ReadItLater v0.3.1
		18: Highlightr v1.2.2
		19: YTranscript v0.8.2
		20: Readwise Official v2.0.1
		21: Excalidraw v1.9.17
		22: Antidote Grammar Checker Integration v2.1.2
		23: Hover Editor v0.11.13
		24: Auto Link Title v1.4.1
		25: Projects v1.16.3
		26: Omnisearch v1.15.2
		27: Multi-Column Markdown v0.8.3
		28: Extract url content v0.12.1
		29: Tracker v1.10.9
		30: Style Settings v1.0.6
		31: Icon Folder v2.1.2
		32: Banners v1.3.3
		33: Editing Toolbar v2.3.1
		34: MySnippets v1.2.3
		35: Quiet Outline v0.3.15
		36: Advanced URI v1.37.0
		37: Homepage v3.5.1
		38: Callout Manager v1.0.1
		39: Metadata Menu v0.5.1
		40: Mind Map v1.1.0
		41: Emoji Magic v0.2.2
		42: Templater v1.16.0
		43: Full Calendar v0.10.7
		44: Day Planner v0.5.8
		45: Calendar v1.5.10
		46: Periodic Notes v0.0.17
		47: Enhancing Mindmap v0.2.5
		48: obsidian markmind v1.7.8
		49: QuickAdd v1.2.1
		50: Global Search and Replace v0.3.0
		51: Obligator v1.2.0
		52: Recent Files v1.3.7
		53: Waypoint v1.4.0
		54: Custom Frames v2.4.6
		55: Text Generator v0.3.20
		56: Smart Connections v1.6.34
		57: Admonition v10.0.1
		58: Kanban v1.5.3
		59: DB Folder v3.4.0

### Plugin version

1.16.3

### Obsidian version

1.37

### OS

macOS

**Comment by Acylation** [comment]
Since the `status` field of the board view is used to categorize notes into board columns, and each note should only belong to one column, only single-value fields, but not arrays, are available for the feature. In your case, you can use below:
```yaml
tags:
  - tag a
  - tag b
aliases:
type:
category:
date: 2023-01-27 03:09:56
status: process
```

Here's the preview:
![image](https://github.com/marcusolsson/obsidian-projects/assets/73122375/d17841e0-a949-4898-b268-35df4b8c177e)

**Comment by Acylation** [comment]
Moving to discussions section since the array fields (or, list properties) are not intended to be recognized by the status option. You can try assigning predefined values to fields following the instructions in #661 . This requires a plaintext field for the status options.
